elyra/metadata/metadata.py

```diff
@@ -441,6 +457,20 @@ def _get_schema(self, schema_name):
 
         return schema_json
 
+    def _validate_metadata(self, name, metadata_json):
+        """Validate the metadata.  If no schema name is present, False is returned.
```

Comment:
minor nit. according to docstring, the method may return False, True, or ValidationError but it looks like it every only returns True or ValidationError

Reply:
Good catch - that was a part of a later change.